spec/loader_spec.rb

```diff
@@ -1,10 +1,57 @@
 require 'spec_helper'
 
 describe ActiveSetting::Loader do
+  let(:config_filename) { 'spec/settings.yml' }
+
   it 'should parse a settings file to get settings' do
-    ActiveSetting::Loader.load_settings('spec/settings.yml')
+    ActiveSetting::Loader.load_settings(config_filename)
     settings = ActiveSetting::Setting.registered_settings
     settings.keys.should include :maximum_percent
   end
+
+  context 'when building a settings hash from the settings file' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.